### ISSUE TITLE
[TCM] Installation guide

### ISSUE THREAD

**Issue by p7nov** [tarantool/doc#3630]
**Product:** Tarantool Cluster Manager
**Since:** TCM 1.0.0, Tarantool 3.0
**Audience/target:** admins
**Root document:** TBD
**SME:** @ filonenko-mikhail 

# Details

A guide on Tarantool Cluster Manager installation/deployment.

- ``tcm`` is a ready-to-run executable
- Supported platforms: Linux and macOS
- TCM is included in the Tarantool Enterprise SDK
- No other distribution ways
- No additional system requirements on top of [Tarantool EE system requirements](https://www.tarantool.io/en/doc/latest/enterprise/setup/#system-requirements)

Bottom line: we need to mention TCM in the Tarantool EE setup guide and give a link from the TCM overview page.
